src/Console/Deployer.php

```diff
@@ -127,26 +127,15 @@ private function generateArchive(SymfonyStyle $io, ProgressBar $progress) : void
 
         // Cache PHP's binary in `.bref/bin/php` to avoid downloading it
         // on every deploy.
-        /*
-         * TODO Allow choosing a PHP version instead of using directly the
-         * constant `PHP_TARGET_VERSION`. That could be done using the `.bref.yml`
-         * config file: there could be an option in that config, for example:
-         * php:
-         *     version: 7.2.2
-         */
+        $phpVersion = $projectConfig['php']['version'] ?? DEFAULT_PHP_TARGET_VERSION;
+
         $progress->setMessage('Downloading PHP in the `.bref/bin/` directory');
         $progress->display();
-        if (!$this->fs->exists('.bref/bin/php/php-' . PHP_TARGET_VERSION . '.tar.gz')) {
+        if (!$this->fs->exists('.bref/bin/php/php-' . $phpVersion . '.tar.gz')) {
             $this->fs->mkdir('.bref/bin/php');
-            $defaultUrl = 'https://s3.amazonaws.com/bref-php/bin/php-' . PHP_TARGET_VERSION . '.tar.gz';
-            /*
-             * TODO This option allows to customize the PHP binary used. It should be documented
-             * and probably moved to a dedicated option like:
-             * php:
-             *     url: 'https://s3.amazonaws.com/...'
-             */
```

Comment:
> I didn't mention the php/url configuration in the README for now, I think it would require more information on how to build the php archive (php binary, ext directory with opcache extension).

No problem, it's a different change so it's better outside this PR.

Can you restore this comment so that we don't forget the TODO?